libraries/persistent_store/src/lib.rs

```diff
@@ -365,6 +365,8 @@ extern crate alloc;
 mod buffer;
 #[cfg(feature = "std")]
 mod driver;
+#[cfg(feature = "std")]
```

Comment:
Does this `std` condition correctly interact with the rest of the code? We use `std` for tests and fuzzing only so far. Therefore, I was wondering if you can just compile the OpenSK library with `std` later for non-testing purposes?